metabase/metabase
# Make collections official via menu

Epic https://github.com/metabase/metabase/issues/22826

How to test:
- Run Metabase EE
- Go to a collection that is not root or personal
- Make sure it is possible to make the collection official/regular via the header menu

Please note that `Edit this collection` menu option will be removed in a follow-up PR

<img width="245" alt="Screenshot 2022-06-24 at 17 38 01" src="https://user-images.githubusercontent.com/8542534/175558369-05b63f18-b86a-433e-b3f3-2c3962877199.png">
